packages/usa-in-page-navigation/src/usa-in-page-navigation.twig

```diff
@@ -9,6 +9,7 @@
     data-scroll-offset="0"
     data-root-margin="0px 0px 0px 0px"
     data-threshold="1"
+    data-minimum-heading-count="3"
```

Comment:
**issue**: Is this needed if the default is `3`?

```suggestion

```

Reply:
I removed this in favor of making it a storybook controlled value per @mahoneycm's suggestion!